layers/exporters/to_excel.py

```diff
@@ -0,0 +1,120 @@
+from openpyxl.comments import Comment
+from openpyxl.styles import PatternFill, Font
+
+try:
+    from core import Layer
+    from exporters import ExcelTemplates
+except ModuleNotFoundError:
+    from ..core import Layer
+    from ..exporters import ExcelTemplates
+
+class NoLayer(Exception):
+    pass
+
+class ToExcel:
+    def __init__(self, domain='enterprise', server=False, local=None):
```

Comment:
I'm not sure this API is solid (and by extension the ExcelTemplates constructor API). 
1. `server` should probably be `use_taxii` or something like that, I don't think `server` alone is a descriptive name.
2. I'm concerned about having separate `server` and `local` arguments. The behavior is unclear if I were to specify `server=True` and also a local file. I'm not sure if we can combine the arguments in any way, maybe something like this:
   `__init__(self, domain, mode="taxii")` (or `"requests"` or `"local"`), and if `local` is specified an additional argument `localstix` is required? If localstix is specified but mode is not `local` you'd need to throw an exception. I _think_ that's an improvement in how understandable the arguments are at a glance, but maybe not.

Reply:
I'm not convinced throwing an error if localstix is set and `local` is not specified is the way to go, but I've swapped the entire system over to a single source parameter which makes things cleaner. It'll throw a `BadSource` error if `local` is specified but no path is provided, which is the most common problem you'd face in that scenario, and the same error if the source is not one of `taxii`, `repo`, or `local`.